lld/MachO/ConcatOutputSection.cpp

```diff
@@ -184,15 +184,42 @@ uint64_t TextOutputSection::estimateStubsInRangeVA(size_t callIdx) const {
     InputSection *isec = inputs[i];
     isecEnd = alignToPowerOf2(isecEnd, isec->align) + isec->getSize();
   }
+
+  // Tally up any thunks that have already been placed that have address higher
+  // than the equivalent callIdx. We first find the index of the first thunk
+  // that is beyond the current inputs[callIdx].
+  auto itPostcallIdxThunks = std::partition_point(
```

Comment:
I think we can use llvm::partition_point()

Reply:
Do you mean another form or it ? We're already using `partition_point`.